examples/AspNetCore/Startup.cs

```diff
@@ -59,91 +56,54 @@ public void ConfigureServices(IServiceCollection services)
-                case "zipkin":
-                    services.AddOpenTelemetryTracing((builder) => builder
-                        .SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(this.Configuration.GetValue<string>("Zipkin:ServiceName")))
-                        .AddAspNetCoreInstrumentation()
-                        .AddHttpClientInstrumentation()
-                        .AddZipkinExporter());
-
-                    services.Configure<ZipkinExporterOptions>(this.Configuration.GetSection("Zipkin"));
-                    break;
-                case "otlp":
-                    // Adding the OtlpExporter creates a GrpcChannel.
-                    // This switch must be set before creating a GrpcChannel when calling an insecure gRPC service.
-                    // See: https://docs.microsoft.com/aspnet/core/grpc/troubleshoot#call-insecure-grpc-services-with-net-core-client
-                    AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
-
-                    services.AddOpenTelemetryTracing((builder) => builder
-                        .SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(this.Configuration.GetValue<string>("Otlp:ServiceName")))
-                        .AddAspNetCoreInstrumentation()
-                        .AddHttpClientInstrumentation()
-                        .AddOtlpExporter(otlpOptions =>
-                        {
-                            otlpOptions.Endpoint = new Uri(this.Configuration.GetValue<string>("Otlp:Endpoint"));
-                        }));
-                    break;
-                default:
-                    services.AddOpenTelemetryTracing((builder) => builder
-                        .AddAspNetCoreInstrumentation()
-                        .AddHttpClientInstrumentation()
-                        .AddConsoleExporter());
-
-                    // For options which can be bound from IConfiguration.
-                    services.Configure<AspNetCoreInstrumentationOptions>(this.Configuration.GetSection("AspNetCoreInstrumentation"));
-
-                    // For options which can be configured from code only.
-                    services.Configure<AspNetCoreInstrumentationOptions>(options =>
-                    {
-                        options.Filter = (req) =>
-                        {
-                            return req.Request.Host != null;
-                        };
-                    });
-
-                    break;
-            }
+            services.AddOpenTelemetryTracing((builder) => builder
+                .SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(serviceName: "aspnetcore", serviceNamespace: "example"))
+                .AddAspNetCoreInstrumentation()
+                .AddConsoleExporter()
+                .AddZipkinExporter(options =>
+                {
+                    var hostname = Environment.GetEnvironmentVariable("ZIPKIN_HOSTNAME") ?? "localhost";
```

Comment:
can we use the official Env variables, and let SDK automatically read it? https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#jaeger-exporter